lib/manageiq/providers/workflows/engine.rb

```diff
@@ -34,13 +34,26 @@ def self.seedable_classes
           %w[ManageIQ::Providers::Workflows]
         end
 
+        def self.floe_runner_name
+          if (runner_setting = Settings.ems.ems_workflows.runner)
```

Comment:
Wonder if we should we add `:runner:` to the `config/settings.yml` so we don't have hidden settings?

Reply:
Agreed - we try to avoid hidden settings.

Reply:
Or maybe we should add to config/environments/development.yml only?

I honestly don't understand the purpose of this setting.  It seems to imply that in dev mode we are not detecting the mode properly, which seems more of a bug to me.

Reply:
This is actually something that I could use, on my workstation I have docker, podman, and kubernetes available to run with and I regularly test `floe` with all three.  It would be nice to override what the provider plugin uses.

I agree though `config/environments/development.yml` since you would never want to use this in production.